server/src/internalClusterTest/java/org/elasticsearch/search/fetch/subphase/InnerHitsIT.java

```diff
@@ -176,7 +176,10 @@ public void testSimpleNested() throws Exception {
         assertThat(innerHits.getAt(0).getHighlightFields().get("comments.message").getFragments()[0].string(),
                 equalTo("<em>fox</em> eat quick"));
         assertThat(innerHits.getAt(0).getExplanation().toString(), containsString("weight(comments.message:fox in"));
-        assertThat(innerHits.getAt(0).getFields().get("comments.message").getValue().toString(), equalTo("fox eat quick"));
+        assertThat(
+            innerHits.getAt(0).getFields().get("comments").getValue(),
+            equalTo(Collections.singletonMap("message", Collections.singletonList("fox eat quick")))
+        );
```

Comment:
This changen in the test is due to the different output format for nested fields now.